### ISSUE TITLE
Splitsen design_decisions.md in ontwerprichtlijnen en architectuurprincipes

### ISSUE THREAD

**Issue by fsamwel** [VNG-Realisatie/Haal-Centraal-common#27]
design_decisions.md bevat nu twee soorten informatie, die geen van beide echt ontwerp**beslissingen** zijn.

Een deel van de inhoud betreft richtlijnen en tips voor een berichtontwerper om tot consistent berichtontwerp te komen.

Een deel van de inhoud betreft architectuurprincipes.

Ik stel voor dat dit dan ook in documenten komt met een naam die duidelijk maakt wat het is, waarvoor het is en voor wie het is: respectievelijk "berichtontwerp_richtlijnen.md" en "architectuurprincipes_Haal_Centraal.md"

**Comment by melsk-r** [comment]
Prima idee, ik ga kijken welke regels in welk document thuishoren. Wat de naamgeving betreft zou ik de woorden in de naam van het eerste document omdraaien en daar ook 'Haal_Centraal' in terug laten komen. Geen 'berichtontwerp_richtlijnen.md' maar 'richtlijnen_berichtontwerp_Haal_Centraal.md'.

**Comment by fsamwel** [comment]
ja, de naam die je voorstelt is prima. als het maar duidelijk is wat het is. Mijn wens is vooral om heel verschillende dingen, gericht op mogelijk verschillende doelgroep, bij elkaar te stoppen in één document

**Comment by melsk-r** [comment]
Ok, ik ga eerst maar even de issues #28 en #29 verwerken en daarna deze splitsing aanbrengen.

**Comment by melsk-r** [comment]
Enkele vraagjes:
* Het opdelen van de design decisions over 'ontwerprichtlijnen' en 'architectuurprincipes' betekent dat we in de id's de string 'DD' vervangen door 'OR' en 'AP'. Aangezien vanuit andere documenten of vanuit discussies naar die id's verwezen moet kunnen worden moeten we voorkomen dat ze steeds wijzigen. We moeten dus wel redelijk zeker zijn van de waarden van de id's. Kunnen we daar vanuit gaan of heb je nog twijfels over de naamgeving van de documenten of kunnen we de id's loskoppelen van de naamgeving van de documenten?
* In de discussie van #29 hebben we het over het verwijderen van Design Decisions. Dat zal ook in de toekomst gaan gebeuren. Ik stel echter voor dat na het uitvoeren van het onderliggende issue het verwijderen van een item betekent dat een id ook verdwijnt. We gaan dan dus niet meer hernummeren. Dat betekent dat nieuwe items het eerstvolgende vrije nummer krijgen toegekend.
T.b.v. een logische indeling van de items kunnen nieuwe items tussen al bestaande items worden gevoegd. De nummering van de items hoeft dus niet netjes door te lopen. Kan je je daarin vinden?
